repose-aggregator/commons/utilities/src/test/scala/org/openrepose/commons/utils/servlet/http/HttpServletRequestWrapperTest.scala

```diff
@@ -93,6 +94,7 @@ class HttpServletRequestWrapperTest extends FunSpec with BeforeAndAfter with Mat
       wrappedRequest.getHeaderNames.asScala.toList should contain theSameElementsAs headerMap.keys.filterNot( _ == "foo")
     }
 
+    //todo: fix this test -- it should not be case sensitive
```

Comment:
i haven't pulled this down yet, because crap connection, but i assumed this test started failing because foo started showing up twice. If that's not the case, meh. also we should always lean towards keeping the original value. In a number of places in your version you are having to do equalsIgnoreCase versus a standard equals. When i don't have a crap connection, i'll add a couple of tests.